sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/SemanticConventions.cs

```diff
@@ -114,6 +114,7 @@ internal static class SemanticConventions
 
         public const string AttributeFaasTrigger = "faas.trigger";
         public const string AttributeFaasExecution = "faas.execution";
+        public const string AttributeFaasColdStart = "faas.coldstart";
```

Comment:
nit: sort alphabetically to make it easier for future maintenance.

Reply:
Whole file needs rearrangement  will get this covered in different PR.